sonar-db/src/main/java/org/sonar/db/permission/PermissionRepository.java

```diff
@@ -134,17 +134,19 @@ private void updateProjectAuthorizationDate(DbSession session, @Nullable Long pr
     }
   }
 
-  public void applyPermissionTemplate(DbSession session, String templateKey, Long resourceId) {
-    PermissionTemplateDto permissionTemplate = dbClient.permissionTemplateDao().selectPermissionTemplateWithPermissions(session, templateKey);
+  public void applyPermissionTemplate(DbSession session, String templateUuid, long resourceId) {
+    PermissionTemplateDto permissionTemplate = dbClient.permissionTemplateDao().selectPermissionTemplateWithPermissions(session, templateUuid);
     updateProjectAuthorizationDate(session, resourceId);
     dbClient.roleDao().removeAllPermissions(session, resourceId);
     List<PermissionTemplateUserDto> usersPermissions = permissionTemplate.getUsersPermissions();
+    //TODO should return an empty list if there's no user permissions
```

Comment:
![INFO](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-info.png) Complete the task associated to this TODO comment. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1135)